src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -100,6 +101,17 @@ sealed trait Task {
 
 object Task {
 
+  object Implicits {
+    import scala.language.implicitConversions
+
+    implicit def tasksToAllPlaced(tasks: Seq[Task]): Seq[Placed] = tasks.map(taskAsPlaced(_))(collection.breakOut)
```

Comment:
I disliked the idea of expanding the Task API. In this case the implicits are wrapped in a lib that must be explicitly imported.